code/script.js

```diff
@@ -1,9 +1,187 @@
+const bigContent = document.getElementById('big-content');
+
+const aboutUsButton = document.getElementById('about-us-button');
+const contactUsButton = document.getElementById('contact-us-button');
+const aboutUsContainer = document.getElementById('about-us');
+const contactUsContainer = document.getElementById('contact');
+
+bookButton.addEventListener('click', () => {
+  bigContent.scrollIntoView({behavior: 'smooth'});
+});
+
+aboutUsButton.addEventListener('click', () => {
+  aboutUsContainer.scrollIntoView({behavior: 'smooth'});
+});
+
+contactUsButton.addEventListener('click', () => {
+  contactUsContainer.scrollIntoView({behavior: 'smooth'});
+});
+
+
+// Accordion questions and answers variables
+
+const accordionQuestionOne = document.getElementById('accordion-question-one');
+const accordionAnswerOne = document.getElementById('accordion-answer-one');
+
+const accordionQuestionTwo = document.getElementById('accordion-question-two');
+const accordionAnswerTwo = document.getElementById('accordion-answer-two');
+
+const accordionQuestionThree = document.getElementById('accordion-question-three');
+const accordionAnswerThree = document.getElementById('accordion-answer-three');
+
+const accordionQuestionFour = document.getElementById('accordion-question-four');
+const accordionAnswerFour = document.getElementById('accordion-answer-four');
+
+const accordionQuestionFive = document.getElementById('accordion-question-five');
+const accordionAnswerFive = document.getElementById('accordion-answer-five');
+
+const accordionQuestionSix = document.getElementById('accordion-question-six');
+const accordionAnswerSix = document.getElementById('accordion-answer-six');
+
+
+// Accordion icons variables
+
+const iconOne = document.getElementById('icon-one');
+const iconTwo = document.getElementById('icon-two');
+const iconThree = document.getElementById('icon-three');
+const iconFour = document.getElementById('icon-four');
+const iconFive = document.getElementById('icon-five');
+const iconSix = document.getElementById('icon-six');
+
+
+// Accordion functions
+
+
+accordionQuestionOne.addEventListener('click', () => {
+  accordionAnswerOne.classList.toggle('accordion-visible');
+  accordionAnswerTwo.classList.remove('accordion-visible');
+  accordionAnswerThree.classList.remove('accordion-visible');
+  accordionAnswerFour.classList.remove('accordion-visible');
+  accordionAnswerFive.classList.remove('accordion-visible');
+  accordionAnswerSix.classList.remove('accordion-visible');
+  if (iconOne.innerHTML === '+') {
+    iconOne.innerHTML = '-'
+  } else {
+    iconOne.innerHTML = '+';
+  };
+  if (iconTwo.innerHTML === '-' || iconThree.innerHTML === '-' || iconFour.innerHTML === '-' || iconFive.innerHTML === '-' || iconSix.innerHTML === '-') {
+    iconTwo.innerHTML = '+';
+    iconThree.innerHTML = '+';
+    iconFour.innerHTML = '+';
+    iconFive.innerHTML = '+';
+    iconSix.innerHTML = '+';
+  };
```

Comment:
We can also style this by the pseudo selector ::after or ::before, then you could include it in the toggling of classes :)